core/commands/root.go

```diff
@@ -69,6 +69,9 @@ TOOL COMMANDS
     commands      List all available commands
 
 Use 'ipfs <command> --help' to learn more about each command.
+
+If you want to operate on a different repository than the default at ~/.ipfs,
+then prefix any ipfs command with IPFS_PATH=/path.
```

Comment:
@sahib This is a great add! But I agree, it is worded a bit strange. How about:

> If you want IPFS to use a different directory locally than the default  `~/.ipfs`, set the `IPFS_PATH` environment variable to point to your preferred repository.

Sahib, do you know how to amend commits?